src/sentry/static/sentry/app/actionCreators/organizations.jsx

```diff
@@ -69,8 +69,8 @@ export function fetchOrganizationsByMember(memberId, {setActive}) {
   });
 
   request.then(data => {
-    if (data.length && setActive) {
-      setActiveOrganization(data[0]);
```

Comment:
Hm... the endpoint is returning all organizations the user is in rather than the organization the member belongs to, so they all need to be loaded into context. I think load is also more accurate than set active, since it adds the organization to a few additional places (ex. the switch organization dropdown).

Reply:
Ugh, on second thought this request is wrong